src/styles/components/_definition.scss

```diff
@@ -47,7 +61,34 @@ $breakpoint-background-image-size: 75rem;
       position: relative;
       font-size: map.get($font-sizes, "sm");
       line-height: map.get($line-heights, "lg"); // approx. 28px/20
-      padding: 2.5rem 8.5625rem;
+    }
+
+    div > ul > li {
+      font-weight: 700;
+
+      > ul li {
```

Comment:
I don't think this is what we want, necessarily. If there are lists in the rich text, then we should preserve their styling as lists.

One thing that I would change is to remove the `list-style-type: none` in `elements/_ul.scss`, since that's way too broad a selector to wipe out the list styling.

The marker styling down below I would want to apply to all list items found in rich text blocks.